utils/formatting-utils.js

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
generally, should the formatting be the actual prefixes:
<10^3 => attoNEAR
<10^6 => femtoNEAR
<10^9 => picoNEAR
> 10^12 => nanoNEAR
< 10^15 => 1 microNEAR
< 10^18 => milliNEAR
rest => NEAR

Reply:
Good question. This version came from validator feedback, but either would work. https://github.com/nearprotocol/stakewars/issues/36